pkg/skel/skel.go

```diff
@@ -93,14 +93,20 @@ func PluginMain(cmdAdd, cmdDel func(_ *CmdArgs) error) {
 	}
 
 	if err != nil {
+		if e, ok := err.(*plugin.Error); ok {
+			// don't wrap Error in Error
+			e.Print()
```

Comment:
OK. Also ideally it would be nice if the Exit() was in a single place that these two paths shared...